src/field_theory/krull_topology.lean

```diff
@@ -283,3 +285,242 @@ begin
 end
 
 end totally_disconnected
+
+section compact
```

Comment:
I think all of this should be namespaced (maybe with `krull_topology`?). That's not currently the case, if I understand the diff correctly.